cmd/config-dir.go

```diff
@@ -41,6 +41,10 @@ const (
 
 	// Private key file for HTTPS.
 	privateKeyFile = "private.key"
+
+	// PKCS#12 certificate file for HTTPS.
+	minioPFXFile = "minio.pfx"
+	minioP12File = "minio.p12"
```

Comment:
p12 and pfx looks like to be the same exact format, I think we can keep only one.